gammapy/spectrum/core.py

```diff
@@ -290,6 +294,14 @@ def obs_id(self):
         return self.on_vector.obs_id
 
     @property
+    def exposure(self):
+        return self.on_vector.exposure
+
+    @property
+    def alpha(self):
```

Comment:
Add one-line description for this property using the format described [here](https://gammapy.readthedocs.io/en/latest/development/howto.html#functions-or-class-methods-that-return-a-single-object).